addons/website_forum/static/tests/tours/website_forum_question.js

```diff
@@ -35,6 +36,44 @@ registry.category("web_tour.tours").add('forum_question', {
         trigger: '.o_popover input.o_select_menu_sticky',
         run: 'edit Tag',
     },
+    {
+        content: "press enter to add tags",
+        trigger: '.o_popover .o_select_menu_sticky',
+        async run(helpers) {
+            await delay(350)
+            await helpers.press("Enter");
+        },
+    },
```

Comment:
Actually, instead of pressing enter, you can click on the "Create option" that appears when typing more than one character. That way you don't have to wait, it is the tour system that does it for you. So, to make the tour more robust, you can replace that part and similar parts below by:
```javascript
{
    trigger: ".o-dropdown-item:contains(Create option)",
    run: "click",
},
```